bindings/alicloud/rocketmq/rocketmq.go

```diff
@@ -90,11 +90,11 @@ func (a *AliCloudRocketMQ) Read(ctx context.Context, handler bindings.Handler) e
 		if topicStr == "" {
 			continue
 		}
-		mqType, mqExpression, topic, err := parseTopic(topicStr)
+		mqType, mqExpression, topic, err1 := parseTopic(topicStr)
```

Comment:
Why was this changed to `err1`?

Reply:
Probably because the linter warning of shadow declaration of err which was declared before. This does need to be a different variable name here.

Reply:
@ItalyPaleAle this might be a better solution

```suggestion
        var mqType, mqExpression string
		mqType, mqExpression, topic, err = parseTopic(topicStr)
```

Reply:
I agree

Reply:
OK, I'll change it.